Arrowgene.Ddon.Database/DdonDatabaseBuilder.cs

```diff
@@ -13,6 +16,8 @@ public static class DdonDatabaseBuilder
         private static readonly ILogger Logger = LogProvider.Logger<Logger>(typeof(DdonDatabaseBuilder));
         private const string DefaultSchemaFile = "Script/schema_sqlite.sql";
 
+        public const uint Version = 2;
```

Comment:
Why do we need this version variable? Can the migration contain this information instead? From a usability point of view, I never want to look at this file. I just want to define my migration file and be done with it.

Reply:
We don't necessarily know what's the target version that the current source code is designed to work with. While it could be safe to just assume the highest number found in the migration strategies is the target version, allowing you to specify a version would allow you to downgrade or give you a bit more flexibility.